ui-v2/tests/unit/controllers/dc/kv/index-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:dc/kv/index', 'Unit | Controller | dc/kv/index', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
+  let controller = this.subject();
+  assert.ok(controller);
+});
```

Comment:
Any reason to keep these controller tests around? Feel like with the controllers you have (in this PR at least) it'd just be testing ember.

Reply:
:) These are just ember's auto generated tests, I'm leaving them in for the moment at least as they have actually helped me catch some typos with import paths and such-like when ember errors haven't been very helpful.

This might be because I've never used ember before and I don't know off the top of my head where certain things are. So, for example, when I've typed an import path wrong without realising, ember gives me an unhelpful error, but the test fails and tells me exactly what I've done wrong and where I've done it. This happens less now I've had a bit more time with ember, but it's still helpful.

Right now its more effort to remove them all than to just leave them for the moment. They aren't doing any harm so I'm happy to just leave them in there.